mage_ai/api/resources/StatusResource.py

```diff
@@ -61,4 +66,35 @@ def collection(self, query, meta, user, **kwargs):
             'project_type': project_type,
             'project_uuid': get_project_uuid(),
         }
+
+        display_format = meta.get('_format')
+        if 'with_activity_details' == display_format:
+            from mage_ai.server.server import latest_user_activity
+            project_schedules = PipelineSchedule.repo_query.all()
+            project_schedule_ids = [schedule.id for schedule in project_schedules]
+            project_pipeline_runs = PipelineRun.query.filter(
+                PipelineRun.pipeline_schedule_id.in_(project_schedule_ids)
+            )
+            sorted_pipeline_runs = project_pipeline_runs.order_by(
+                PipelineRun.updated_at.desc()
+            )
+            if sorted_pipeline_runs.count() > 0:
+                last_scheduler_activity = sorted_pipeline_runs[0].updated_at
+            active_pipeline_run_count = project_pipeline_runs.filter(
+                PipelineRun.status.in_(
+                    [
+                        PipelineRun.PipelineRunStatus.INITIAL,
```

Comment:
The INITIAL PipelineRun in the inactive trigger won't run.

Reply:
ok i just changed it to running